horace_core/sqw/file_io/class_helpers/is_sqw_struct.m

```diff
@@ -2,7 +2,11 @@
 % Check if input structure can be considered an sqw structure.
 %
 if ~isstruct(input)
-    is = false;
+    if isa(input,'binfile_v4_block_tester') % this is for testing new file format
+        is = true;
+    else
```

Comment:
its rather compatibility function necessary in the past when old style classes were used, but still useful to identify correct input for sqw object or some object to use instead of sqw object in tests.